index.js

```diff
@@ -31,45 +37,116 @@ client.on('messageCreate', async message => {
                 //Limit the amount of SPAM to the channel
                 matchCount++;
                 if (matchCount > MAX_RESPONSES){
-                    message.reply(`You added too many searches to your message.  Please limit the number of card search requests for each message to ${MAX_RESPONSES}`);
+                    message.reply(`You added too many requests to your message.  Please limit the number of requests for each message to ${MAX_RESPONSES}`);
                     return; //This will break out of the loop and function, might not be the best approach, but it's the one we can take here.
                 }
+
                 const commandInput = match[1];
                 //console.log(`Command Input: "${commandInput}"`);
-                //Perform a better regex check here for data sanitization on searchTerm
+                //Perform a regex check here for data sanitization on commandInput
                 //  - a-zA-Z (),|-!' - (space is included)
+                const commandMatches = commandInput.match(/^([aAtTiIvV]\:)?([a-zA-Z \(\),\|\-!\']+)$/);
+
+                //console.log(`Matches ${commandMatches}`);
+
+                if(commandMatches){
+                    //console.log(`commandMatches[1]: "${commandMatches[1]}"`);
+
+                    var defaultView = commandMatches[1] === undefined || commandMatches[1] === '';
+                    var textView = commandMatches[1] && commandMatches[1].toLowerCase() === "t:";
+                    var imageView = commandMatches[1] && commandMatches[1].toLowerCase() === "i:";
+                    var verboseView = commandMatches[1] && commandMatches[1].toLowerCase() === "v:";
+
+                    //[[card]] [[t:card]] [[i:card]] [[v:card]]
+                    var cardSearch = defaultView || textView || imageView || verboseView;
```

Comment:
These can all be `const`s since they don't change.
```suggestion
                    const defaultView = commandMatches[1] === undefined || commandMatches[1] === '';
                    const textView = commandMatches[1] && commandMatches[1].toLowerCase() === "t:";
                    const imageView = commandMatches[1] && commandMatches[1].toLowerCase() === "i:";
                    const verboseView = commandMatches[1] && commandMatches[1].toLowerCase() === "v:";

                    //[[card]] [[t:card]] [[i:card]] [[v:card]]
                    const cardSearch = defaultView || textView || imageView || verboseView;
```

Reply:
OK with this as well.